### PR TITLE
Corrected spelling and word flow

### PR DESCRIPTION
**Pull request by martendegroot**
### Description

Small set of spelling and readability corrections for the Aria roles page.

### Motivation

I was giving the page a thorough read and was distracted one or two incomplete words and long sentences. So I decided to open up this PR with an attempt to rectify those points.

### CODE REVIEW

File: files/en-us/web/accessibility/aria/roles/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ There are 6 categories of ARIA roles:
 
 ### 1. Document structure roles
 
-Document Structure roles are used to provide a structural description for a section of content. Most of these roles should no longer be used as browsers now support semantic HTML element with the same meaning. The roles without HTML equivalents, such as presentation, toolbar and tooltip roles, provide information on the document structure to assistive technologies such as screen readers as equivalent native HTML tags are not available.
+Document Structure roles are used to provide a structural description for a section of content. Most of these roles should no longer be used as browsers now support semantic HTML elements with the same meaning. The roles without equivalent HTML element, such as presentation, toolbar and tooltip roles, provide information on the document structure to assistive technologies like screen readers as equivalent native HTML tags are not available.
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/aria/roles/index.md:25]: This is not an improvement.

File: files/en-us/web/accessibility/aria/roles/index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ These are included for completeness, but in most cases are rarely, if ever, usef
 
 ### 2. Widget roles
 
-The various widget role are used to define common interactive patterns. Similar to the document structure roles, some of these roles duplicate the semantics of native HTML elements that are well supported, and should not be used. The difference between the two lists is that, generally, the widget roles require JavaScript interaction and the document structure roles don't necessarily.
+Widget roles are used to define common interactive patterns. Like document structure roles, some widget roles overlap with the semantics of native HTML elements that are well-supported and therefore should be avoided. The key difference is that widget roles typically require JavaScript for interaction, while document structure roles often do not.
```
Comment (hamishwillee) [inline files/en-us/web/accessibility/aria/roles/index.md:76]:
```suggestion
Widget roles are used to define common interactive patterns. Like document structure roles, some widget roles have the same semantics as well-supported native HTML elements, and therefore should be avoided. The key difference is that widget roles typically require JavaScript for interaction, while document structure roles often do not.
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much.

File: files/en-us/web/accessibility/aria/roles/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/accessibility/aria/roles/index.md]: @hamishwillee "element -> elements" is a grammatical fix, right? "Support HTML element" is a grammatical error without a/the/plural.

**Review by zhowan2020 — COMMENTED** [review]
Done

File: files/en-us/web/accessibility/aria/roles/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/accessibility/aria/roles/index.md]: Thanks @Josh-Cena. Yes, an "s" there would be correct grammar. I've take the original text and just fixed that in https://github.com/mdn/content/pull/35800

FWIW I scanned this as an unnecessary "matter of taste" change - apologies @martendegroot

**Pull request merged**